### PR TITLE
Operational panels cypress fix 

### LINKED ISSUE #201: [Cypress Tests] Update cypress tests for Operational Panels
1. Fix failing tests and falky tests 
2. Add more tests for new code changes.

### PR DESCRIPTION
**Pull request by ps48**
### Description
Updated flaky cypress tests for panels 

### Issues Resolved
#201 

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: .eslintcache [hunk 1]
```diff
@@ -0,0 +1 @@
...11638,"mtime":1675271951923,"results":"10","hashOfConfig":"7"},{"filePath":"11","messages":"12","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"1i553ik",{"filePath":"13","messages":"14","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"15","messages":"16","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"17","messages":"18","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"/Users/sgguruda/work/git_codebase/osd-3.0/OpenSearch-Dashboards/plugins/dashboards-observability/public/components/custom_panels/custom_panel_table.tsx",[],"/Users/sgguruda/work/git_codebase/osd-3.0/OpenSearch-Dashboards/plugins/dashboards-observability/public/components/custom_panels/custom_panel_view.tsx",[],"/Users/sgguruda/work/git_codebase/osd-3.0/OpenSearch-Dashboards/plugins/dashboards-observability/public/components/custom_panels/helpers/add_visualization_popover.tsx",[],"/Users/sgguruda/work/git_codebase/osd-3.0/OpenSearch-Dashboards/plugins/dashboards-observability/public/components/custom_panels/panel_modules/visualization_flyout/visualization_flyout.tsx",[]]
```
Comment (mengweieric) [inline .eslintcache]: Should we remove this?

Comment (ps48) [reply .eslintcache]: I think you're looking at an older commit. It is already removed in the next one https://github.com/opensearch-project/dashboards-observability/pull/244/commits/c584a93f0748c2ff6f0d2783fa102e9ae27601e7

**Review by mengweieric — APPROVED** [review]

File: public/components/custom_panels/panel_modules/visualization_container/visualization_container.tsx [hunk 2]
```diff
@@ -93,7 +93,6 @@ export const VisualizationContainer = ({
   spanParam,
 }: Props) => {
   const [isPopoverOpen, setIsPopoverOpen] = useState(false);
-  const [disablePopover, setDisablePopover] = useState(false);
```
Comment (derek-ho) [inline public/components/custom_panels/panel_modules/visualization_container/visualization_container.tsx:96]: nice

**Review by derek-ho — APPROVED** [review]
clean

**Pull request merged**